cms/src/test/resources/test-data/api-specifications/ApiSpecSyncFromApigeeJobIntegrationTest/oasV3_complete.html

```diff
@@ -207,193 +178,69 @@ <h6>Body</h6>
   } ],
   "birthDate" : "2010-10-22"
 }</code></pre></div>
-
-
-
-
-            <div class="body__schema__header">Schema
-                <span class="schema__header__buttons">
-                                                                        <button class='schemaButton js collapseAll' onclick="collapseAll('07aecf8a-dd08-423c-a9c0-0df76bc1d26c')">Collapse all</button>
-                                                                        <button class='schemaButton js expandAll' onclick="expandAll('07aecf8a-dd08-423c-a9c0-0df76bc1d26c')">Expand all</button>
+                                                                <div class="body__schema__header">Schema
+                                                                    <span class="schema__header__buttons">
+                                                                        <button class='schemaButton js collapseAll' onclick="collapseAll('a7b833e4-f260-46f5-901e-2da15b11e5fd')">Collapse all</button>
+                                                                        <button class='schemaButton js expandAll' onclick="expandAll('a7b833e4-f260-46f5-901e-2da15b11e5fd')">Expand all</button>
                                                                     </span>
+                                                                </div>
+                                                                <div class="body__schema" data-schema-uuid="a7b833e4-f260-46f5-901e-2da15b11e5fd">
+                                                                    <table data-disablesort="true">
```

Comment:
Tables should be wrapped in `nhsd-m-table`.

Reply:
To be addressed in [ADZ-1194](https://nhsd-jira.digital.nhs.uk/browse/ADZ-1194).